webapp/TargetedMS/js/BaseQCPlotPanel.js

```diff
@@ -200,6 +200,26 @@ Ext4.define('LABKEY.targetedms.BaseQCPlotPanel', {
         });
     },
 
+    queryQCInstruments: function(successCallback, callbackScope) {
+        LABKEY.Query.executeSql({
+            schemaName: 'targetedms',
+            sql: 'SELECT DISTINCT instrumentSerialNumber FROM samplefile',
```

Comment:
Let's include the model too, since people are more likely to recognize 'Q Exactive' than a potentially obscure serial number.